File: Punctuality Incentive .java
import java.util.*;
public class Hello {

    public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
	    int startAmount= sc.nextInt();
	    int dayCount = sc.nextInt();
	    int payableAmount=0;
        int counter=0;
        while(counter < dayCount){
            payableAmount+= (startAmount + counter*200);
            counter++;
        }
        
        System.out.print(payableAmount);

	}
}

File: Punctuality Incentive.c
#include<stdio.h>
#include <stdlib.h>

int main()
{
    int startAmount,dayCount;
    scanf("%d%d",&startAmount,&dayCount);
    int payableAmount=0;
    int counter=0;
    while(counter < dayCount){
        payableAmount+= (startAmount + counter*200);
        counter++;
    }
    
    printf("%d",payableAmount);

}
